sdk/core/Azure.Core/src/Shared/LruCache.cs

```diff
@@ -43,7 +46,7 @@ public bool TryGet(TKey key, out TValue value)
                     return true;
                 }
 
-                value = default(TValue);
+                value = default(TValue)!;
```

Comment:
If TValue is a reference type, then wouldn't this be null?

Reply:
Ah I thought the ! syntax just meant "it's fine if this is null"